packages/lib-classifier/src/store/TutorialStore.js

```diff
@@ -140,14 +135,19 @@ const TutorialStore = types
     function * fetchTutorials () {
       const workflow = getRoot(self).workflows.active
       const tutorialsClient = getRoot(self).client.tutorials
+      const upp = getRoot(self).userProjectPreferences
 
       self.loadingState = asyncStates.loading
       try {
         const response = yield tutorialsClient.get({ workflowId: workflow.id })
         const { tutorials } = response.body
         if (tutorials && tutorials.length > 0) {
-          tutorials.forEach(tutorial => self.fetchMedia(tutorial))
+          yield Promise.all(tutorials.map(tutorial => self.fetchMedia(tutorial)))
```

Comment:
I don't think `fetchMedia` is called anywhere else in the code so it can be a private method.
```suggestion
          yield Promise.all(tutorials.map(tutorial => flow(fetchMedia)(tutorial)))
```